scripts/linters/pre_commit_linter.py

```diff
@@ -436,7 +436,8 @@ def _get_task_output(all_messages, task, semaphore):
         semaphore: threading.Semaphore. The object that controls how many tasks
             can run at any time.
     """
-    all_messages += task.output
+    if task.output:
```

Comment:
I think we don't need `if` here as `task.output` will always return a list and if it is returning `None` then it is a sign that an exception has been raised somewhere in the linter and linter has failed. @kevinlee12 What do you think?

Reply:
I think that sounds reasonable.

Reply:
I don't see any exceptions being logged after adding this, but the linter fails without it, so I'm not really sure what to make of it. Seems we need more debug messages in the linter processes?

Reply:
I don't see any exceptions being logged after adding this, but the linter fails without it, so there aren't any good clues for why this is necessary.

My instinct tells me there's a branch of logic in the linter somewhere that leads to a bare `return`, or that there's a branch that never hits a `return` at all.

Reply:
I ran linter on your code locally without `if` statement and all checks are passing successfully. Could you please try to run linter one more time without `if` statement and if the linter failed then it is fine to use `if` here.

Reply:
Yup seems to work now, not sure what has changed between versions. Thanks for confirming!